### PR TITLE
Fix npm install failure on Windows

### PR DESCRIPTION
**Pull request by halitogunc**
see: #1422

cc @mor-n4

### CODE REVIEW

File: n4js-libs/packages/n4js-cli/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "internalVersion": "2019-03-15T08:46:12Z",
   "description": "N4JS compiler and command line tools.",
   "scripts": {
-    "postinstall": "./npm-postinstall.js",
+    "postinstall": "node ./npm-postinstall.js",
     "build": "./npm-build.sh",
```
Comment (mor-n4) [inline n4js-libs/packages/n4js-cli/package.json]: And what about this shell script `npm-build.sh`? Does it work on Windows?
Also `n4jsc.js` a few lines further down: doesn't that one need to be adjusted, as well?

File: n4js-libs/packages/n4js-cli/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "internalVersion": "2019-03-15T08:46:12Z",
   "description": "N4JS compiler and command line tools.",
   "scripts": {
-    "postinstall": "./npm-postinstall.js",
+    "postinstall": "node ./npm-postinstall.js",
```
Comment (mor-n4) [inline n4js-libs/packages/n4js-cli/package.json]: Have you tested that it still runs on macOS?

File: n4js-libs/packages/n4js-cli/package.json (hunk 1 above)
Comment (mor-n4) [inline n4js-libs/packages/n4js-cli/package.json]: Have you checked whether other package.json files contain similar cases? (shell scripts, .js-files launched via shebang)

File: n4js-libs/packages/n4js-cli/package.json (hunk 2 above)
Comment (halitogunc) [reply n4js-libs/packages/n4js-cli/package.json]: yes, it still works on macOS

File: n4js-libs/packages/n4js-cli/package.json (hunk 1 above)
Comment (halitogunc) [reply n4js-libs/packages/n4js-cli/package.json]: Windows does not have a built-in support for .sh files. They need to install a third-party tool like Cygwin, Git bash or similar.

Comment (halitogunc) [reply n4js-libs/packages/n4js-cli/package.json]: yes, I have checked it and could not find any

**Comment by mor-n4** [comment]
Builds green, merging ...

**Pull request merged**